lib/src/special_inline_span_base.dart

```diff
@@ -4,7 +4,7 @@ import 'package:flutter/rendering.dart';
 ///  create by zmtzawqlp on 2019/7/10
 ///
 
-abstract class SpecialInlineSpanBase {
+mixin SpecialInlineSpanBase {
```

Comment:
Can you explain more about why this should be `mixin` instead?

Reply:
> Can you explain more about why this should be `mixin` instead?
It is used as a mixin in the ExtendedWidgetSpan component, and it reports an error in dart3, asking for a mixin